packages/kolibri-tools/lib/webpack.config.plugin.js

```diff
@@ -172,11 +167,27 @@ module.exports = (
 
   if (isCoreBundle && mode === 'production') {
     bundle.plugins.push(
-      // requires >= v3.0.0, which is specified in the kolibri-core yarn workspace
+      // requires >= v3.0.0, which is specified in the kolibri package
       new webpack.NormalModuleReplacementPlugin(/^vue-intl$/, 'vue-intl/dist/vue-intl.prod.min.js'),
     );
   }
 
+  if (isCoreBundle) {
+    bundle.plugins.push(
+      // We need to do this to sidestep the janky @vue/composition-api package's index.js which
+      // conditionally imports the prod or dev version of the package based on the NODE_ENV, but
+      // only exposes the common JS builds, and reassigns them to the module.exports, thereby
+      // preventing webpack from importing it consistently in both how we use it internally
+      // in the core bundle, and also then access it via externals in the plugin bundles.
```

Comment:
I am following this comment 80% and I think I understand in broad strokes what is happening, but if I were to really explain the _why_, I don't think I would get it. The two phrases that I keep getting tripped up on are 
> only exposes the common JS builds, and reassigns them to the module.exports

and
> also then access it via externals in the plugin bundles

Reply:
Basically, internally, the composition api does something really janky for its exports, which breaks how we expose the 'core' modules on the default frontend bundle for the plugins to reuse without having to bundle them. This is a workaround for that.

If it makes you feel any better, once we've upgraded to Vue 2.7, we can get rid of this entirely, because we won't need the plugin any more!